recipes/imagej/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "imagej" %}
```

Comment:
Should we name this package `python-imagej` in case we get the real `imagej` at some point?

Reply:
Ping @ctrueden : do you think we should still call the package `imagej` or something such as `pyimage` or `imagej.py`? Would be nice to have the same name as the pypi package.

Reply:
I would like to keep the name as `imagej`. It is currently the only way to get ImageJ from conda, and is very likely to remain the primary supported way until conda's Java packaging support is fully ironed out (see conda-forge/conda-forge.github.io#590). Once that happens, we can update the `imagej` conda package to work differently as appropriate. But personally I see no urgency; the `jgo`-based solution seems very powerful to me, allowing easy distribution/installation of any Java-based component you want in a lightweight way, reusing Java's already-mature Maven-based dependency management.

Reply:
I feel like you're right @bgruening but couldn't we imagine shipping Fiji more than ImageJ in the future?

The desktop application ImageJ is never used anyway and 99% of the users download Fiji. So maybe keeping this package name to imagej and consider creating a Conda Fiji package is fine?

Reply:
> The desktop application ImageJ is never used anyway

It would be used more often if I fixed the situation with the Java-8 update site. :-/

> couldn't we imagine shipping Fiji more than ImageJ in the future?

Current longer-term plan is to make Fiji an add-on. You install ImageJ, then enable Fiji. Maybe ImageJ prompts which update sites you want the first time you launch.

> So maybe keeping this package name to imagej and consider creating a Conda Fiji package is fine?

I think it would be confusing if the `imagej` conda package did not ship JARs, but the `fiji` package did ship JARs. Let's be consistent. See my longer comment in the main PR thread.